types/victory/index.d.ts

```diff
@@ -327,6 +327,112 @@ declare module 'victory' {
         y: number;
     };
 
+    export interface VictoryClipContainerProps {
```

Comment:
Maybe it's missing `extends VictoryContainerProps` due to [common container props](https://formidable.com/open-source/victory/docs/common-container-props)? Then this interface should only have the props specific to it. WDYT @Theomg

Reply:
I don't think `VictoryClipContainer` has anything to do with other containers though.
The code(https://github.com/FormidableLabs/victory-core/blob/master/src/victory-clip-container/victory-clip-container.js#L10) makes no references to the `VictoryContainer` used as a base for other containers such as `VictoryVoronoiContainer` (https://github.com/FormidableLabs/victory/blob/master/packages/victory-voronoi-container/src/victory-voronoi-container.js#L196), which is where they get the `VictoryContainerProps` from.
It would deduplicate `children`, `className` and `events`, but I'm unsure if it would really be the correct thing to do here. Do you think it's worth doing it anyways?

Reply:
I agree with not extending the whole `VictoryContainerProps` interface, but since you mention it would reuse `children`, `className` and `events` maybe we could use [pick](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktk).

``` typescript
... extends Pick<VictoryContainerProps, "children" | "className" | "events">
```
WDYT?

Reply:
I was going by the types in the documentation, but it seems like in the actual type definitions neither `VictoryContainerProps` nor `VictoryCommonProps` have `className` or `children`, and other types like `VictoryLabelProps` implement them themselves. `children` could be left to React.Component but would lose its specificity and its JSDoc.
Only `events` can be picked from `VictoryContainerProps`, but I personally wouldn't use that as the component itself doesn't inherit from it, but if you think it's worth it adding the `Pick<VictoryContainerProps, 'events'>` I could add it. Do you think it's better to add the Pick with only one property?

Reply:
:thinking: I don't think we should use `Pick` for just one prop, I think we should go with `DOMAttributes` once again.

Reply:
Alright, already using the `DOMAttributes` on the events.
I believe everything is according to your suggestions then.